pkg/test/componentlist.go

```diff
@@ -0,0 +1,55 @@
+package test
+
+import (
+	"github.com/stretchr/testify/require"
+	"gopkg.in/yaml.v3"
+	"io"
+	"net/http"
+	"testing"
+)
+
+const (
+	kymaComponentListURL = "https://raw.githubusercontent.com/kyma-project/kyma/main/installation/resources/components.yaml"
+	kymaNamespace        = "kyma-system"
+)
+
+type KymaComponentList struct {
+	DefaultNamespace string `yaml:"defaultNamespace" json:"defaultNamespace"`
+	Prerequisites    []Component
+	Components       []Component
+}
+
+type ComponentList struct {
```

Comment:
Is it used somewhere?

Reply:
Right, thanks for pointing it out! That's dead code...will drop it